include/algebra_vector.h

```diff
@@ -330,7 +330,7 @@ c_float OSQPVectorf_dot_prod_signed(const OSQPVectorf *a,
                              const OSQPVectorf *b);
 
 /* check l <= u elementwise */
-c_int OSQPVectorf_ew_lt_eq(OSQPVectorf *l, OSQPVectorf* u);
+c_int OSQPVectorf_ew_leq(OSQPVectorf *l, OSQPVectorf* u);
```

Comment:
I changed the name of this function to resemble the latex command for `<=`.

Reply:
Maybe this should be `OSQPVectorf_all_leq`, since it's behaviour is not returning an elementwise comparison as output in in a vector.

Reply:
I agree. Can you please make the edit?

Reply:
Also, `OSQPVectorf_ew_bound_vec` could be renamed to something like `OSQPVectorf_clip_vec`.

Reply:
> I agree. Can you please make the edit?

Done.